### PR TITLE
London class 7/omar sharif - HTML-CSS - Week 2

### PR DESCRIPTION
**Pull request by omarsharif-gr**
# Your Details

- Your Name: Omar Sharif
- Your City: London
- Your Slack Name: Omar Sharif

# Homework Details

- Module: HTML-CSS
- Week: 2

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,19 +1,157 @@
...
+  background-color: white;
+  position: sticky;
+}
+.navbar .logo {
+  height: 60px;
+  margin-left: 30px;
+  float: left;
+}
+.navbar a{
+  float: left;
```
Comment (LucyMac) [inline css/style.css:19]: Avoid using `float`. Use flexbox instead for layout.

File: css/style.css [hunk 2]
```diff
@@ -1,19 +1,157 @@
...
+  color: grey;
+  text-align: center;
+  padding: 14px 20px;
+  background-color: white;
+}
+.navbar a:hover{
+  color: orange;
+}
+.right {
+  float: right;
```
Comment (LucyMac) [inline css/style.css:31]: Same comment on using flexbox instead of float.

File: css/style.css [hunk 3]
```diff
@@ -1,19 +1,157 @@
...
+  width: 100%;
+  height: auto;
+  padding:115px;
+  margin-top: 0px;
+}
+.main .refill{
+  width: 100%;
+  height: auto;
+  padding:115px;
+  margin-top: 0px;
```
Comment (LucyMac) [inline css/style.css:108]: Remember, the purpose of CSS classes is that they are re-usable! 
Don't repeat your code here. Only define these styles once for one class (for example `.main-icon`) and then apply that class to all 3 elements in your HTML.

File: index.html [hunk 4]
```diff
@@ -7,15 +7,77 @@
...
+                <a class="icon"><img src="img\menu-hamburger.svg" alt="Menu icon"> </a>
+                
+                    <div class="right">
+                            <a href="#" class="current-tab">Meet Karma</a>
+                            <a href="#">How it works</a>
+                            <a href="#">Store</a>
+                            <a href="#">Blog</a>
+                            <a href="#">Help</a>
+                            <a href="#">Login</a>
+                    </div>
```
Comment (LucyMac) [inline index.html:28]: Could this be a list instead?

File: index.html [hunk 5]
```diff
@@ -7,15 +7,77 @@
...
+            <h1>Introducing Karma</h1>
+            <h3>Bring WIFI with you, everywhere you go</h3>
+            <div>
+                <button type="button" class="button">Learn More</button>
+            </div>
+        </div>
+    </section>
+    <div class="statement">
+        <h1>Everyone needs a little Karma.</h1>
+    </div>
```
Comment (LucyMac) [inline index.html:43]: This `h1` should be inside the next section (and the `div` that wraps it is not necessary) as its the heading for the 3 large orange icons.

File: index.html [hunk 6]
```diff
@@ -7,15 +7,77 @@
...
+                Internet for all devices
+            </h3>
+        </div>
+        <div class="coffee">
+            <img src="img\icon-coffee.svg" alt="coffee mug icon">
+            <h3 class="statement-for-icons">
+                Boost your productivity
+            </h3>
+        </div>
+        <div class = "refill">
```
Comment (LucyMac) [inline index.html:57]: As I mentioned in your CSS, these 3 icons should be sharing the same CSS class.

File: index.html [hunk 7]
```diff
@@ -7,15 +7,77 @@
...
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+<hr class="solid">
 
 </body>
+<footer>
+    <h3 class="statement"> Join us on</h3>
+        <section class="footer">
+        <div class="twitter-icon">
+            <img src="img\twitter-icon.svg" alt="Twitter">
```
Comment (LucyMac) [inline index.html:72]: Prefer forward slashes `/` inside paths

File: index.html [hunk 8]
```diff
@@ -7,15 +7,77 @@
...
+        </div>
+        <div class="facebook-icon">
+            <img src="img\facebook-icon.svg" alt="Facebook">
+        </div>
+        <div class="instagram-icon">
+            <img src="img\instagram-icon.svg" alt="Instagram">
+        </div>
+        </section>
+    <h4 class="statement"> © Karma Mobility Inc</h4>
```
Comment (LucyMac) [inline index.html:81]: This is probably not a heading since it's a tiny bit of text. You can use `p` or `span` for this one.

File: css/style.css [hunk 9]
```diff
@@ -1,19 +1,157 @@
...
+  .navbar a:not(:nth-child(0)) {display: none;}
+  .navbar a.icon {
+    float: right;
+    display: block;
+  }
+  .main{
+    display: flex;
+    flex-direction: column;
+    margin-left: 0px;
+    margin-right: 250px;
```
Comment (LucyMac) [inline css/style.css:146]: This margin-right is likely to cause you trouble. Try and keep everything balanced if you want elements to be centered and re-position themselves in a reliable way when you resize the screen.
Giving appropriate sizes to your icons and using Flexbox on the container should help you do this nicely.

File: css/style.css [hunk 10]
```diff
@@ -1,19 +1,157 @@
...
+.intro h3{
+  font-size: 20px;
+  margin-top: 20px;
+  font-weight: 100;
+}
+
+/*Button - general styles*/
+.button{
+  background-color: orange;
+  color: white;
```
Comment (LucyMac) [inline css/style.css:75]: To follow the design a little more closely, consider adding padding inside the button, as well as giving it a border-radius.

File: css/style.css [hunk 11]
```diff
@@ -1,19 +1,157 @@
...
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+body, footer {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+/*Navigation*/
+.navbar{
+  background-color: white;
+  position: sticky;
```
Comment (LucyMac) [inline css/style.css:11]: Does this rule work for you? It isn't working for me. Have you tried using `position: fixed` instead? 
```
    position: fixed;
    width: 100%;
```

**Review by LucyMac — COMMENTED** [review]
Nice start and good attempt at making the page responsive. 
Try and use flexbox more, it'll help you with positioning ([look here](https://www.freecodecamp.org/news/an-animated-guide-to-flexbox-d280cf6afc35/) or ask in class if you're not sure how to do this)
I wasn't able to see your background-image - have you tried adding dots in front of the relative path? --> `url(../img/first-background.jpg);` This means 'go up one level' and fixes this issue where the CSS file can't access the image because it's in a higher folder.

File: css/style.css (hunk 1 above)
Comment (omarsharif-gr) [reply css/style.css:19]: Updated - commits made

File: css/style.css (hunk 3 above)
Comment (omarsharif-gr) [reply css/style.css]: Updated CSS

File: index.html (hunk 4 above)
Comment (omarsharif-gr) [reply index.html:28]: Updated HTML and CSS

File: index.html (hunk 8 above)
Comment (omarsharif-gr) [reply index.html:81]: Updated HTML

File: css/style.css (hunk 9 above)
Comment (omarsharif-gr) [reply css/style.css:151]: CSS updated to use flexbox

File: index.html (hunk 5 above)
Comment (omarsharif-gr) [reply index.html]: Resolved

File: css/style.css (hunk 10 above)
Comment (omarsharif-gr) [reply css/style.css:90]: Updated CSS to meet this

File: index.html (hunk 7 above)
Comment (omarsharif-gr) [reply index.html]: Updated my CSS to meet this.

**Pull request closed without merging**